aws/table_aws_account_contact.go

```diff
@@ -0,0 +1,163 @@
+				Description: "The third line of the primary contact address, if any.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.AddressLine3"),
+			},
+			{
+				Name:        "company_name",
+				Description: "The name of the company associated with the primary contact information, if any.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.CompanyName"),
+			},
+			{
+				Name:        "contact_account_id",
+				Description: "Account ID to get contact details for.",
+				Type:        proto.ColumnType_STRING,
+				// Transform:   transform.FromQual("contact_account_id"),
+			},
+			{
+				Name:        "city",
+				Description: "The city of the primary contact address.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.City"),
+			},
+			{
+				Name:        "country_code",
+				Description: "The ISO-3166 two-letter country code for the primary contact address.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.CountryCode"),
+			},
+			{
+				Name:        "district_or_county",
+				Description: "The district or county of the primary contact address, if any.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.DistrictOrCounty"),
+			},
+			{
+				Name:        "phone_number",
+				Description: "The phone number of the primary contact information. The number will be validated and, in some countries, checked for activation.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.PhoneNumber"),
+			},
+			{
+				Name:        "postal_code",
+				Description: "The postal code of the primary contact address.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.PostalCode"),
+			},
+			{
+				Name:        "state_or_region",
+				Description: "The state or region of the primary contact address. This field is required in selected countries.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.StateOrRegion"),
+			},
+			{
+				Name:        "website_url",
+				Description: "The URL of the website associated with the primary contact information, if any.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContactInformation.WebsiteUrl"),
+			},
+		}),
+	}
+}
+
+type accountContactData = struct {
+	ContactInformation types.ContactInformation
+	ContactAccountId    *string
+}
+
+//// LIST FUNCTION
+
+func getAwsAccountContact(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+
+	// Create Session
+	svc, err := AccountClient(ctx, d)
+	if err != nil {
+		logger.Error("aws_account_contact.getAwsAccountContact", "service_creation_error", err)
```

Comment:
Please remember to update logging messages like these once the function name is updated too